packages/fiori/src/ShellBar.ts

```diff
@@ -159,12 +150,10 @@ const HANDLE_RESIZE_DEBOUNCE_RATE = 200; // ms
  * <h3>ES6 Module Import</h3>
  * <code>import "@ui5/webcomponents-fiori/dist/ShellBar";</code>
  *
+ * @csspart root - Used to style the outermost wrapper of the <code>ui5-shellbar</code>
+ *
  * @constructor
- * @author SAP SE
- * @alias sap.ui.webc.fiori.ShellBar
- * @extends sap.ui.webc.base.UI5Element
- * @tagname ui5-shellbar
- * @appenddocs sap.ui.webc.fiori.ShellBarItem
+ * @extends UI5Element
```

Comment:
Should the slots be defined on a class level now?

e.g.
```
* Class description showcase
*
* @slot {ShowcaseType[]} default - Unnamed slot description showcase
*
```

Reply:
I believe this rule apply only for unnamed slots:
https://github.com/SAP/ui5-webcomponents/issues/7610#unnamed-slot

Reply:
As @plamenivanov91 mentioned it related to unnamed slots (slots without accessor)